io.osdn.simplesok.yaml

```diff
@@ -0,0 +1,34 @@
+app-id: io.osdn.simplesok
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: simplesok
+finish-args:
+  - --device=all
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=home
+modules:
+  - name: simplesok
+    buildsystem: simple
+    build-commands:
+      - sed -i 's/usr\/share\/simplesok/app\/share\/simplesok/g' skin.c
+      - make
+      - install -Dm755 simplesok /app/bin/simplesok
+      - mkdir -p /app/share/simplesok
+      - mv skins /app/share/simplesok/skins
+      - install -D io.osdn.simplesok.metainfo.xml /app/share/metainfo/io.osdn.simplesok.metainfo.xml
+      - install -D io.osdn.simplesok.desktop /app/share/applications/io.osdn.simplesok.desktop
+      - install -D io.osdn.simplesok.svg /app/share/icons/hicolor/scalable/apps/io.osdn.simplesok.svg
+    sources:
+      - type: archive
+        url: http://simplesok.osdn.io/simplesok-1.0.3.tar.xz
+        sha256: cbfaa6b1befc907652b9420220376377a43e9668131dd18573c7caa9678bc610
+      - type: file
```

Comment:
have these files been submitted upstream? It's a bug then don't ship any.

Reply:
even since the tarball ship what is needed for rpm and deb. It's all generic for Linux desktop.

Reply:
I didn't understand your first comment -- are you saying I should wait until this is merged upstream and a new tarball is created, before continuing?

Reply:
no, but having it in the next release would be nice.